libImaging/Resample.c

```diff
@@ -184,7 +184,6 @@ precompute_coeffs(int inSize, int outSize, struct filter *filterp,
                 } else if (x == xmax - 1) {
                     xmax -= 1;
                 }
-                continue;
             }
             k[x] = w;
```

Comment:
Hmm, but such uninitialized values should be outside of `xmax`. I need 10 minutes to investigate why this leads to the bug.